experimental/saiswitchextensions.h

```diff
@@ -37,6 +37,17 @@ typedef enum _sai_switch_attr_extensions_t
 {
     SAI_SWITCH_ATTR_EXTENSIONS_RANGE_START = SAI_SWITCH_ATTR_END,
 
+    /**
+     * @brief Set Switch ICMP_ECHO session state change event notification callback function passed to the adapter.
+     *
+     * Use sai_icmp_echo_session_state_change_notification_fn as notification function.
+     *
+     * @type sai_pointer_t sai_icmp_echo_session_state_change_notification_fn
+     * @flags CREATE_AND_SET
+     * @default NULL
+     */
+    SAI_SWITCH_ATTR_ICMP_ECHO_SESSION_STATE_CHANGE_NOTIFY,
```

Comment:
you should add  = SAI_SWITCH_ATTR_EXTENSIONS_RANGE_START

Reply:
that fixed the issue. thank you!